lib/index.js

```diff
@@ -17,6 +17,25 @@ var shell = require('shelljs');
 var restify = require('restify');
 
 /**
+ * checks if the path is device rooted, that is, if it has [drive letter]:\ at its beggining.
+ * @private
+ * @param {String} fpath
+ *      The file path to check
+ * @returns {Boolean}
+ *      Whether the path is rooted or not.
+ */
+function isWin32DeviceRoot(fpath) {
+    // Only for win32
+    if (process.platform !== 'win32') {
```

Comment:
I tested it; It returns "linux", as  you can see from the screenshot below. 

![image](https://cloud.githubusercontent.com/assets/1890900/17175749/ac76b30e-53df-11e6-8259-8fe4a29fd2e8.png)

I guess that's the correct behavior since bash is emulating a Linux filesystem. As you can see from the next screenshot, the drive letter is case sensitive. 

![image](https://cloud.githubusercontent.com/assets/1890900/17175792/e84702da-53df-11e6-8611-03beca2b83b5.png)